keras_imaging/preprocessing/transforms.py

```diff
@@ -0,0 +1,123 @@
+import numpy
+import skimage.transform
+from scipy.ndimage.filters import gaussian_filter
+from scipy.ndimage.interpolation import map_coordinates
+
+"""
+Note: Standardization and transforms assumes that x comes in WxHxC format from the reader
+"""
+
+
+def horizontal_flip(prob):
+    assert 0. < prob < 1.
+
+    def f(x):
+        if numpy.random.random() < prob:
+            return numpy.fliplr(x)
+        return x
+
+    return f
+
+
+def vertical_flip(prob):
+    assert 0. < prob < 1.
+
+    def f(x):
+        if numpy.random.random() < prob:
+            return numpy.flipud(x)
+        return x
+
+    return f
+
+
+def rotate90(prob):
+    assert 0. < prob < 1.
+
+    def f(x):
+        if numpy.random.random() < prob:
+            return numpy.rot90(x, 2, axes=(0, 1))
+        return x
+
+    return f
+
+
+def rescale(scale, **kwargs):
+    """
+    Rescales the image according to the scale ratio.
+    :param scale: The scalar to rescale the image by.
+    :param kwargs: Additional arguments for skimage.transform.resize.
+    :return: The rescale function.
+    """
+
+    axes_scale = (scale, scale, 1.0)
+
+    return lambda x: skimage.transform.resize(x, numpy.multiply(x.shape, axes_scale), **kwargs)
```

Comment:
While I personally prefer returning a lambda, we should match the nested function style used by the other functions for consistency.